optional_plugins/varianter_yaml_to_mux/avocado_varianter_yaml_to_mux/mux.py

```diff
@@ -294,7 +294,7 @@ def iteritems(self):
 
     def items(self):
         """ Slower implementation with the use of __getitem__ """
-        return self.iteritems()
+        return self.iteritems()  # pylint: disable=W1620
```

Comment:
By the way, after reviewing the modifications after @willianrampazzo's comment, we might me able to remove this method entirety, as ``ValueDict`` inherits from ``dict`` and the return from ``dict.items`` is already an iterator.